filters/builtin/static.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2015 Zalando SE
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package builtin
+
+import (
+	"fmt"
+	"github.com/zalando/skipper/filters"
+	"net/http"
+	"path"
+)
+
+type static struct {
+	webRoot, root string
+}
+
+// Returns a filter Spec to serve static content from a file system
+// location. Marks the request as served.
+//
+// Filter instances of this specification expect two parameters: a
+// request path prefix and a local directory path. When processing a
+// request, it clips the prefix from the request path, and appends the
+// rest of the path to the directory path. Then, it uses the resulting
+// path to serve static content from the file system.
+//
+// Name: "static".
+func NewStatic() filters.Spec { return &static{} }
```

Comment:
Maybe this filter should expect the parameters in the reverse order and make the web root prefix optional.

For ex.,
`Any() -> static("/some/path") ...`, would serve https://server/images/file.jpg from /some/path/images/file.jpg
`Any() -> static("/some/path", "/static") ...`, would still serve https://server/static/images/file.jpg from /some/path/images/file.jpg

This would make it behave more like the usual `DocumentRoot` setting